gigadb/app/tools/readme-generator/tests/_data/golden_readme_100039.txt

```diff
@@ -0,0 +1,51 @@
+[DOI]
+10.5524/100039
+
+[Title]
+Genomic data of the Puerto Rican Parrot (<em>Amazona vittata</em>) from a locally funded project.
+
+[Release Date]
+2012-09-11
+
+[Citation]
+Oleksyk TK; Guiblet W; Pombert JF; Valentin R; Martinez-Cruzado JC (2012): Genomic data of the Puerto Rican Parrot (<em>Amazona vittata</em>) from a locally funded project.
+GigaScience Database. https://dx.doi.org/10.5524/100039
```

Comment:
whilst you're working on the readme, can you remove this spureous carriage return, the "GigaScience Database" line should continue on the end of the title in the line above.